lib/rspec/sidekiq/matchers/be_retryable.rb

```diff
@@ -11,7 +11,8 @@ def initialize(expected_retry)
         end
 
         def description
-          if @expected_retry.is_a?(Fixnum)
+          num_class = 1.class # Ruby 2.4 returns Integer, older returns Fixnum
+          if @expected_retry.is_a?(num_class)
```

Comment:
I would say you don't need to assign `1.class` to a var, just use `Integer` or `Numeric` instead of `Fixnum`.

the difference is just that Float is not an Integer but is a Numeric

also, squash your small patch into a single commit, although github is now able to do this on merge